### PR TITLE
[13.0][MIG] auth api key

### PR DESCRIPTION
**Pull request by Laurent-Corron**

### CODE REVIEW

File: auth_api_key/tests/test_auth_api_key.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+            self.env["auth.api.key"]._retrieve_uid_from_api_key("dummy")
+
+    def test_wrong_key(self):
+        with self.assertRaises(ValidationError), self.env.cr.savepoint():
+            self.env["auth.api.key"]._retrieve_uid_from_api_key("api_wrong_key")
+
+    def test_user_not_allowed(self):
+        # only system users can check for key
+        with self.assertRaises(AccessError), self.env.cr.savepoint():
+            self.env["auth.api.key"].sudo(
```
Comment (lmignon) [inline auth_api_key/tests/test_auth_api_key.py]: @Laurent-Corron To change the user in v13 you must use `with_user`

**Review by lmignon — CHANGES_REQUESTED** [review]

**Review by lmignon — APPROVED** [review]
Thank you @Laurent-Corron (code review)

**Review by rousseldenis — APPROVED** [review]
Code review

**Comment by xiaoxin1991** [comment]
odoo13 keycloak?

**Pull request merged**